packages/react-core/src/helpers/DashboardWrapper.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
```

Comment:
I love this class! Can it move to `packages/react-core/src/demos/DashboardWrapper.js`? Having it exported from `react-core` for consumers is strange...

Yes, you'll have to lose the types until we support TS examples, sorry!

Reply:
Ah I agree, it is a bit weird to get the file from `react-core` since it's just demo code. I'll move it and tweak it to a `js` file.

Reply:
Was also getting circular dependencies having it in helpers so this should resolve that as well